lxd/network/acl/acl_ovn.go

```diff
@@ -1067,14 +1069,30 @@ func ovnParseLogEntry(input string, prefix string) string {
 	}
 
 	// Filter for our ACL.
-	if !strings.HasPrefix(aclEntry["name"], prefix) {
+	if prefix != "" && !strings.HasPrefix(aclEntry["name"], prefix) {
```

Comment:
should this change be in a separate commit? Its not clear what its fixing?

Reply:
Not needed anymore. Fixed